CDC-Data-Reconciliation-Frontend/src/components/Report.jsx

```diff
@@ -149,6 +149,19 @@ export default function Report({ reportID }) {
       id: "TotalWrongAttributes",
       footer: (props) => props.column.id,
     },
+    {
+      header: "Case Class Status",
+      accessorFn: (row) => row.CaseClassStatus.toString(),
+      cell: (info) => (
+        <span
+          className='text-blue-500 cursor-pointer underline'
+        >
+          {info.getValue()}
+        </span>
+      ),
+      id: "CaseClassStatus",
+      footer: (props) => props.column.id,
+    },
```

Comment:
You added this as a column to the stats table but when you click on the stats table, you get redirected to a blank screen. Since you made this stat clickable and filterable, you need to modify the switch case that starts on line 335.

Reply:
The CaseClassStatus should be on the Discrepancy table not the stats right?

Reply:
Doesn't make sense for it to be in the stats table